tamr_unify_client/dataset/resource.py

```diff
@@ -237,6 +238,20 @@ def from_geo_features(self, features, geo_attr=None):
             self._features_to_updates(features, record_id, key_attrs, geo_attr)
         )
 
+    def upstream_datasets(self):
+        """The Dataset's upstream datasets.
+
+        :return: A list of the Dataset's upstream datasets.
+        :rtype: list[:class:`~tamr_unify_client.dataset.DatasetURI`])
```

Comment:
Extra parentheses on this comment, and should be `dataset.uri.DatasetURI`--build the docs and verify that links work as expected